linux/debian/mozillavpn.postrm

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+case "$1" in
+remove)
```

Comment:
It is perhaps food for thought about what do on other `postrm` events, such as an upgrade. But I think this is good to cover this particular issue.